api/constants/constants.go

```diff
@@ -72,7 +72,7 @@ const (
 	Github = "github"
 
 	// HumanDateFormatSeconds is a human readable date formatting with seconds
-	HumanDateFormatSeconds = "Jan _2 15:04:05 UTC"
+	HumanDateFormatSeconds = "Jan _2 2006 15:04:05 UTC"
```

Comment:
It's worth noting this is going to change how dates are formatted in several other places, have we checked all of those make sense to show the year ?

Reply:
It's used in 3 places, `tctl status`, `tctl top` and `tsh db ls -v`.  The last, `tsh db ls -v` has a PR #23316 to remove it due to the variable no longer being used.  I've confirmed both `tctl status` and `tctl top` as these are displays for users, not for data processing.